src/components/Courses/components/CourseCard/components/DisplayCourses.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const DisplayCourses = ({ course, authors }) => {
+  let courseAuthorsNames = "";
+  if (course && course.authors) {
+    courseAuthorsNames = course.authors
+      .filter((authorId) => authorId !== undefined)
```

Comment:
A neat shortcut for such filtering would be `.filter(Boolean)`. This basically runs the `Boolean(authorId)` callback for each filtered item and does the same check but more neatly.